packages/api/core/src/api/package.ts

```diff
@@ -37,13 +37,19 @@ type DoneFunction = (err?: Error) => void;
 type PromisifiedHookFunction = (buildPath: string, electronVersion: string, platform: string, arch: string) => Promise<void>;
 type PromisifiedFinalizePackageTargetsHookFunction = (targets: TargetDefinition[]) => Promise<void>;
 
+function hidePromiseFromPromisfy<P extends unknown[]>(fn: (...args: P) => Promise<void>): (...args: P) => void {
```

Comment:
```suggestion
function hidePromiseFromPromisify<P extends unknown[]>(fn: (...args: P) => Promise<void>): (...args: P) => void {
```

typo in fn name

can we also mark this as `@deprecated` with a comment to remove this once we fix the issue upstream in packager and release a new major version?